extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -177,6 +177,10 @@ export class AmpIframe extends AMP.BaseElement {
 
     /** @private {!IntersectionObserver} */
     this.intersectionObserver_ = null;
+
+    if (!this.element.hasAttribute('frameborder')) {
+      this.element.setAttribute('frameborder', '0');
```

Comment:
But setting it on a `<amp-iframe>` is useless?

> Same way setting resizable sets scrolling=no on the actual amp-iframe rather than iframe

Hello bug!